tasks/task457_matres_conditional_classification.json

```diff
@@ -8,6 +8,9 @@
     "Categories": [
         "Classification"
```

Comment:
Classification -> Verification

Reply:
This task doesn't sound to me like a verification task. It sounds more like a standard binary classification task. (Otherwise, we should always assign a binary classification task under verification: "verify if it is the positive class".)

Reply:
I'm not sure! I assigned binary classification tasks under the verification category.